src/impl/tpls/KokkosBlas1_iamax_tpl_spec_avail.hpp

```diff
@@ -107,18 +107,26 @@ Kokkos::View<const SCALAR*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
 #if defined (KOKKOSKERNELS_INST_DOUBLE)
 KOKKOSBLAS1_IAMAX_TPL_SPEC_AVAIL_CUBLAS( unsigned long, double,                 Kokkos::LayoutLeft, Kokkos::CudaSpace)
 KOKKOSBLAS1_IAMAX_TPL_SPEC_AVAIL_CUBLAS( unsigned int, double,                  Kokkos::LayoutLeft, Kokkos::CudaSpace)
+KOKKOSBLAS1_IAMAX_TPL_SPEC_AVAIL_CUBLAS( unsigned long, double,                 Kokkos::LayoutLeft, Kokkos::CudaUVMSpace)
```

Comment:
I think we will want to guard this with a `KOKKOS_UVM_ENABLED` ifdef?

Reply:
@jjwilke Yes. Do we already have ```KOKKOS_UVM_ENABLED``` in Kokkos or Kokkos Kernels? or will you add it to CMake?

Reply:
Yeah, it comes from Kokkos itself. It's `KOKKOS_ENABLE_CUDA_UVM`, sorry, had the wrong name. As long as we include `Kokkos_Core.hpp` we will have that macro.

Reply:
@jjwilke One question: what is the difference between ```KOKKOSKERNELS_INST_MEMSPACE_CUDAUVMSPACE``` and ```KOKKOS_UVM_ENABLED```?

Reply:
Thanks, @jjwilke